src/components/uncommon/BuyDagForm/index.tsx

```diff
@@ -550,6 +554,13 @@ export const TransactionReceipt: React.FC<TransactionReceiptProp> = ({
   //     .then((value) => setBalance(value));
   // }, [loading]);
 
+  const DAG_EXPLORER_SEARCH =
+    "http://lb.exchanges.constellationnetwork.io:9000/transaction/";
+
+  const handleOpenLink = (txHash: string) => {
+    window.open(`${DAG_EXPLORER_SEARCH}${txHash}`, "_blank");
```

Comment:
Need to check which network wallet is connected to. If it's TEST (ceres), then use lb.exchange .... otherwise use dagexplorer.